src/sentry/api/urls.py

```diff
@@ -601,6 +603,17 @@
                     OrganizationAlertRuleTriggerActionIndexEndpoint.as_view(),
                     name="sentry-api-0-organization-alert-rules-trigger-actions",
                 ),
+                # Data Export
+                url(
+                    r"^(?P<organization_slug>[^\/]+)/data-export/$",
+                    DataExportEndpoint.as_view(),
+                    name="sentry-api-0-organization-data-export",
+                ),
+                url(
+                    r"^(?P<organization_slug>[^\/]+)/data-export/(?P<data_id>[^\/]+)/$",
```

Comment:
this might be personal preference but I feel like it should be `data_export_id` to clarify, maybe just for me `data_id` seems so general like it could be anything and it takes a hot second to get the context.

Reply:
totally, makes sense